src/schema/movie/movie.resolvers.ts

```diff
@@ -0,0 +1,36 @@
+import { Resolvers } from '../../generated/graphql';
+import { DEFAULT_CONTENTFUL_LIMIT } from '../../utils/contentful';
+import getAllMovies from '../../models/Movie/getAll';
+import { CONTENTFUL_INCLUDE } from '../../types/contentful';
+import getMovieById from '../../models/Movie/getById';
+
+export const movieResolvers: Resolvers = {
+	Query: {
+		movie: async (_parent, { id }) => {
+			const movie = await getMovieById(id, {
+				include: CONTENTFUL_INCLUDE.genresWithMovies,
```

Comment:
We probably just want to use `CONTENTFUL_INCLUDE.genres` here, when querying for the movie(s), we don't want to know the other movies in the same genre of this movie we're querying.

Reply:
Yeah, I had doubts here; "how deep should I go with it?"  I agree to stop at genres

Reply:
Done!

![image](https://github.com/thisdot/movies-api/assets/39612839/266785f7-8618-47aa-9b92-2188c2e67a45)